website/docs/r/api_gateway_stage.html.markdown

```diff
@@ -78,6 +79,15 @@ The following arguments are supported:
 * `variables` - (Optional) A map that defines the stage variables
 * `tags` - (Optional) A mapping of tags to assign to the resource.
 
+### Nested Blocks
+
+#### `access_log_settings`
+
+* `destination_arn` - (Required) ARN of the log group to send the logs to.
+~> **Note:** Referencing a log group arn from a Terraform resource will normally include a `:*` suffix which is not valid for the destination arn, Terraform automatically strips this suffix when configuring this argument.
```

Comment:
I don't think we need this bold callout that makes it sound like Terraform is doing something wrong or this is Terraform specific behavior (as the ARN value with `:*` coming from the CloudWatch API). I think a simple ``Automatically removes trailing `:*` if present.`` in the line above would suffice. 👍